knowledge-base/grid-state-filter-nullable-bool.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Programatically filter a nullable bool column in the Grid by null values
+description: Learn how to programmatically filter nullable bool values in the Grid component for Blazor.
+type: how-to
+page_title: How to filter a Grid column bound to a nullable bool by the null values programatically
+slug: grid-kb-state-filter-nullable-bool
+tags: grid, blazor, column, datagrid, filter, state, nullable, bool
+res_type: kb
+ticketid: 1658561
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor</td>
```

Comment:
Perhaps the KB is applicable to the TreeList too?

Reply:
It does